tests/webtests/Manage/Client/FakeEntityChangeRequestClient.php

```diff
@@ -25,8 +25,12 @@
 
 class FakeEntityChangeRequestClient implements EntityChangeRequestRepository
 {
-    public function openChangeRequest(ManageEntity $entity, ?ManageEntity $pristineEntity, Contact $contact): array
-    {
+    public function openChangeRequest(
+        ManageEntity $entity,
+        ?ManageEntity $pristineEntity,
+        Contact $contact,
+        string $jiraTicketNumber
+    ): array {
```

Comment:
See 
SlevomatCodingStandard.Arrays.TrailingArrayComma
SlevomatCodingStandard.Functions.RequireTrailingCommaInCall

Reply:
Updated them throughout. Could be a nice default sniff!